config/hooks/pre-commit

```diff
@@ -1,58 +0,0 @@
-#!/usr/bin/env python
```

Comment:
annoyingly I think we still need this thing since we haven't ported the js checks over to `.pre-commit-config.yaml`

Reply:
hmm... i'll look into getting that done before this PR then

Reply:
lol been wanting to do that since 2021 apparently https://github.com/getsentry/sentry/issues/25291